stitches/fx_stitch.py

```diff
@@ -269,6 +269,10 @@ def gridded_stitching(out_dir, rp):
             for i in rslt.keys():
                 ds = rslt[i]
                 ds = ds.sortby('time').copy()
+                rp_csv = (out_dir + '/' + "stitched_" + ds[i].attrs['model'] + '_' + ds[i].attrs['variable'] +
+                          '_' + single_id + "_recipe.csv")
+                ds[i].attrs['recipe'] = rp_csv
```

Comment:
would it be too annoying to make this 'recipe_location' or 'recipe_address' instead of just 'recipe'?